packages/output/src/utils/backgroundAudio.tsx

```diff
@@ -45,7 +41,7 @@ function BackgroundAudio({ backgroundAudio, id }) {
   return (
     <amp-story-grid-layer template="fill">
       <amp-video
-        autoPlay="autoplay"
+        autoplay="autoplay"
```

Comment:
Maybe double check that this is correct on the frontend.

Reply:
We checked and updated casing are correct!

We can simply go ahead even without specifying value for [`autoplay` attribute](https://amp.dev/documentation/examples/components/amp-video/#autoplay).
```suggestion
        autoplay
```

Reply:
I'll need to check whether that doesn't cause any AMP validation errors because this is all going through React first, which likes to adjust attributes a little bit.

Reply:
Yeah when I use just `autoplay` or `autoPlay` it gets removed by the sanitizer.